thinc/shims/tensorflow.py

```diff
@@ -1,9 +1,19 @@
-from typing import List, Optional, Dict
 import contextlib
+import copy
 import itertools
+import os
+import shutil
+import tempfile
 from io import BytesIO
+from typing import Any, Callable, Dict, List, Optional, TypeVar
```

Comment:
Worst nitpick, but I always try to make `from typing` the first import.

Actually, speaking of imports: maybe we should adpot [`isort`](https://github.com/timothycrosley/isort) (just realised this is by the author of Hug btw) and have the import sorting taken care of automatically? I found this config which is apparently compatible with `black`: https://github.com/psf/black/issues/333#issuecomment-516088368